### PR TITLE
Add - Issue and PR templates

### PR DESCRIPTION
**Pull request by fabiankaegy**
This PR adds two Issue Templates and a Pull Request Template to the Repository.

#11

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+What process did you follow to verify that your change has the desired effects?
+
+- How did you verify that all new functionality works as expected?
+- How did you verify that all changed functionality works as expected?
+- How did you verify that the change has not introduced any regressions?
+
+Describe the actions you performed (e.g., commands you ran, text you typed, buttons you clicked) and describe the results you observed.
+-->
+
+### Checklist:
```
Comment (timwright12) [inline .github/PULL_REQUEST_TEMPLATE.md:37]: @fabiankaegy can you update this to the new version?

**Comment by timwright12** [comment]
@fabiankaegy this is throwing a conflict now

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (fabiankaegy) [reply .github/PULL_REQUEST_TEMPLATE.md:37]: Just did  👍

**Comment by timwright12** [comment]
@fabiankaegy changes are fine, but there are some weird conflicts here. Probably just from al lthe recent merges

**Comment by timwright12** [comment]
@fabiankaegy conflicts

**Comment by timwright12** [comment]
@fabiankaegy  ping on this one

**Pull request closed without merging**